src/styles/oeb/oeb_styles.scss

```diff
@@ -164,6 +164,18 @@
 	border-radius: 10px;
 }
 
+/* Hide arrows from input number - Firefox */
```

Comment:
I wonder if it really makes sense to hide the arrows from all input fields. Wasn't the requirement just for the ZIP code? E.g. for the house number I feel it kinda makes sense to have the arrows. Maybe it would make more sense to have a specific CSS class to hide the arrows? Like `forminput-hide-arrows` or sth like that?

Reply:
I updated it to hide arrows only from zip code input.